python/tank/platform/__init__.py

```diff
@@ -10,7 +10,7 @@
 
 
 # Engine management
-from .engine import start_engine, current_engine, get_engine_path, find_app_settings
+from .engine import start_engine, current_engine, get_engine_path, find_app_settings, _restart_engine
```

Comment:
That's what I suspected. I guess I can/will have to adjust this when I merge the code with 0.18, so that's fine. Because I love merging....